opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -917,9 +918,13 @@ def make_model_config_json(
 
     # private methods
     def __qryrem(self, x):
-        # for removing the "QRY:" token
-        y = x.passages.split(" QRY:")[0].split("<|startoftext|>")
-        return y[1]
+        # for removing the "QRY:" token if they exist in passages
+        if "QRY:" in x.passages and "<|startoftext|>" in x.passages:
+            y = x.passages.split(" QRY:")[0].split("<|startoftext|>")
+            return y[1]
+        else:
+            print("QRY: and <|startoftext|> not found in passages. ")
```

Comment:
Why we need to remove "QRY:" and "<|startoftext|>" ?

Message should be ?
```suggestion
            print("QRY: or <|startoftext|> not found in passages. ")
```

Is it possible that this line will print out too many messages if thousands lines doesn't have "QRY"?